src/lib/ext-listen.js

```diff
@@ -0,0 +1,29 @@
+export async function getAllExtensions() {
+  const extensions = await browser.management.getAll();
+  return extensions.filter((extension) => extension.type !== 'theme');
```

Comment:
Personally I would do the opposite and only keep the entries with extension.type == 'extension'.

Reply:
done :-)